test/setup/main.tf

```diff
@@ -14,14 +14,23 @@
  * limitations under the License.
  */
 
+resource "random_id" "folder-rand" {
+  byte_length = 2
+}
+
+resource "google_folder" "ci-iam-folder" {
+  display_name = "ci-tests-iam-folder-${random_id.folder-rand.hex}"
```

Comment:
Can we rename the folder to `ci-cloud-functions` to specify that the projects inside are all for this blueprint
```suggestion
resource "google_folder" "ci-iam-folder" {
  display_name = "ci-cloud-functions-${random_id.folder-rand.hex}"
```